zulip/zulip/__init__.py

```diff
@@ -859,6 +859,38 @@ def create_user(self, request=None):
             request=request,
         )
 
+    def update_state(self, request):
+        # type: (Dict[str, Any]) -> Dict[str, Any]
+        '''
+            Example usage:
+
+            >>> client.update_state({'state': {"entry 1": "value 1", "entry 2": "value 2", "entry 3": "value 3"}})
+            >>> client.get_state({'keys': ["entry 1", "entry 3"]})
+            {'result': 'success', 'state': {'entry 1': 'value 1', 'entry 3': 'value 3'}, 'msg': ''}
+        '''
+        return self.call_endpoint(
+            url='user_state',
+            method='PUT',
+            request=request,
+        )
+
+    def get_state(self, request=None):
```

Comment:
Let's call these e.g. `update_bot_storage`.  We might also want to rename the endpoints to use the `user_storage` name, given what we're calling this feature in the client-side code...